spec/models/storage_spec.rb

```diff
@@ -505,4 +505,24 @@
       expect(storage.smartstate_analysis_count_for_host_id(host.id)).to eq(2)
     end
   end
+
+  context '#storage_clusters' do
+    it 'returns only parents of type storage_cluster' do
+      # A storage mounted on different VCs will have multiple parents
+      storage          = FactoryGirl.create(:storage,               :name => 'test_storage_vmfs', :store_type => 'VMFS')
```

Comment:
ok, I just saw that :storage_vmware is exactly the `:store_type => 'VMFS'`
For the test :+1: to be as generic as possible :smile: